akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,70 @@
+jQuery(document).ready(function ($) {
+
+  var latestInSeries = {
+    "2.2": {
+      latest: "2.2.5",
+      deprecated: true,
+      instead: "2.3"
+    },
+    "2.3": {
+      latest: "2.3.13"
+    },
+    "2.4": {
+      latest: "2.4.0-RC1"
```

Comment:
Yeah, it'd be great to build into the doc-server. Maybe this idea should be delayed until then hm